src/routes/balance-router.js

```diff
@@ -1,20 +1,21 @@
 import express from 'express';
 import balanceController from '../controllers/balance-controller';
-
+import authorization from '../controllers/authorization';
 
 const router = express.Router();
 
 router.route('/')
+  .get(authorization.authorizeAdmin)
   .get((req, res) => {
-    balanceController.getBalanceList()
+    balanceController.findUsers(req.query.name, Number.parseInt(req.query.page, 10))
```

Comment:
parseInt is base 10 by default

Reply:
In some browsers, when string starts with '0', parseInt base is 8, so I prefer always write base 10

Reply:
True, but in this case we work in a controlled environment (NodeJS)